src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -457,6 +457,14 @@ const parseObjectKeyValueToMongoObjectKeyValue = (restKey, restValue, schema) =>
     );
   }
   value = mapValues(restValue, transformInteriorValue);
+
+  if (
```

Comment:
Could you explain this part? This seems to be hacking the MongoDB driver?

Reply:
I follow the line used by @dblythy here https://github.com/parse-community/parse-server/pull/8560#discussion_r1204953867

Reply:
Got it. The reason the PR wasn't merged was because it wasn't fully clear if that was a breaking change. As @dblythy noted that some types are not supported and may need to be added in the future.

Reply:
Not sure what the implication of this is, https://github.com/parse-community/parse-server/pull/8560#discussion_r1222366763 is still a unanswered. But since this is an open topic for months without moving ahead, I believe we should take the risk and merge it into Parse Server 7. Since Parse Server 7 will be a major release, developers are supposed to test their app thoroughly before upgrading anyway.